uldk.py

```diff
@@ -9,75 +9,33 @@ def __init__(self):
         self.filteredPowiatDict = {}
         self.filteredGminaDict = {}
 
+    @staticmethod
+    def fetch_unit_dict(url):
```

Comment:
Fakt, lepiej to wygląda zamiast robić to samo tylko ze zmienionym jednym argumentem

Reply:
Ale może coś tutaj brakuje w porównaniu do poprzedniej wersji tej funkcji chociażby z _fetchGminaDict, bo jednak tam było wydobywanych więcej zmeinnych naraz

Reply:
tak, było pobieranych więcej informacji, ale niepotrzebnie, bo teraz wystarczy na teryt: nazwa